README.md

```diff
@@ -23,15 +23,14 @@ or trademarks, whether by implication, estoppel or otherwise.
 
 ## Contributing
 
-This is the repository for Windows 10 IoT **documentation** hosted at [https://docs.microsoft.com/windows/iot-core](https://docs.microsoft.com/windows/iot-core).
+This is the repository for published [Windows for IoT documentation](https://docs.microsoft.com/windows/iot-core)(.
 
-If you would like to see new coverage or have feedback, please consider [**contributing**](/CONTRIBUTING.md).  You can edit the existing content, add new content, or simply create new [issues](https://github.com/MicrosoftDocs/windows-iotcore-docs/issues). We’ll take a look at your suggestions and will work together to incorporate them into the docs.
+If you would like to see new coverage or have feedback, please consider [**contributing**](https://docs.microsoft.com/CONTRIBUTING).  You can edit the existing content, add new content, or simply create new [issues](https://github.com/MicrosoftDocs/windows-iotcore-docs/issues). We’ll take a look at your suggestions and will work together to incorporate them into the docs.
```

Comment:
Does this page exist? i dont think this path is correct. It is supposed to link to the Contributing.md file in the same directory.

Reply:
Fixed, thanks for the feedback.